### PR TITLE
Add epd7in5b_v2 support

### PR DESCRIPTION
**Pull request by peckpeck**
Support of 7.5" (b) v2 (3 colors with back, white, red)
It is based on  epd7in5_v2 which doesn't work (epd7in5_v3 neither)
It is also based on the arduino C driver and datasheet.
It works on my V3 but the C driver and the datasheet haven't changed between v2 and v3.

### CODE REVIEW

File: src/epd7in5b_v2/command.rs [hunk 1]
```diff
@@ -0,0 +1,156 @@
+//! SPI Commands for the Waveshare 7.5"(B) V2 E-Ink Display
```
Comment (caemor) [inline src/epd7in5b_v2/command.rs:1]:
```suggestion
//! SPI Commands for the Waveshare 7.5"(B) V2 and V3 E-Ink Display
```

Maybe include the v3 as well then if its already tested and working with the same commands.

**Review by caemor — CHANGES_REQUESTED** [review]
Could you please add the display(s) to the changelog and the readme as well?

**Comment by peckpeck** [comment]
Done

**Review by caemor — APPROVED** [review]
Thanks

**Pull request merged**